flutter-studio/src/io/flutter/utils/AddToAppUtils.java

```diff
@@ -5,7 +5,7 @@
  */
 package io.flutter.utils;
 
-import static com.android.tools.idea.gradle.project.importing.GradleProjectImporter.ANDROID_PROJECT_TYPE;
+//import static com.android.tools.idea.gradle.project.importing.GradleProjectImporter.ANDROID_PROJECT_TYPE;
```

Comment:
Is this commented out import intentional?

Reply:
Yes, it broke the canary version. I didn't delete it yet because I'm thinking of asking Android Studio to make it public again.